_posts/2017/2017-02-20-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,207 @@
+The point of using the same technique for both of these things is that it becomes easier to manage and document differences between them in a joint effort. This continually provides value for the company, because setup times for developers are reduced.  
+Using playbooks to install a development environment is a good way of incorporating new developers. Invariably occurring problems are an opportunity to familiarize with the environment and getting used to the workflow by documenting their solution.
+
+### Deployment Playbooks
+The other kind of playbooks would be named `deploy-<project>.yml`.  
+These are meant to execute continuously, deploying changing versions of an application.  
+This type of playbook would typically be executed during a CI/CD run, and a failure during that phase could be the cause for investigating the playbook in the first place.
+
+Let's look at an example playbook (`deploy-unity.yml`):
+{% highlight yml %}
+---
+- hosts: application-vms
+
+  remote_user: deploy
+  become: true
+
+  pre_tasks:
+    - name: Abort on missing ePages-installation
+      shell: rpm -q epages
+      args:
+        warn: false
+      register: check_epages
+      failed_when: "check_epages.rc != 0"
+      changed_when: false
+
+  roles:
+    - role: docker
+      tags: [ docker ]
+
+    - role: epages-unity
+      tags: [ unity ]
+{% endhighlight %}
+This playbook should be fairly self-explanatory, given you know what "[Unity](https://www.epages.com/de/now/)" is.    
+Looking at the pre-task, its intention seems to be to document a dependency, providing a bit more context for this playbook.  
+The roles that are being assigned are `docker` (which would correctly be assumed to install Docker on the host) and `epages-unity`.  
+The installation of Docker in a deployment playbook is questionable, and should probably be refactored once the way of doing the deployment has stabilized (see below).
+Concerning the hosts (`application-vms` in this case), the word implies a group of hosts. This could also be read as "the hosts that this playbook is meant to run against", with the distinction, that depending on the environment, it might only be executed against a single host at a time, using the `--limit` parameter of `ansible-playbook`.  
+Motivation for doing it like this is that the exact same playbooks can be used for different environments / inventories (see below).
+
+## Roles
+While playbooks are the entry point, one will soon be looking at [Roles](http://docs.ansible.com/ansible/playbooks_roles.html), because playbooks should not contain single tasks, but only sets of roles (that are applied to groups of hosts).  
+When learning about a role, the best place to start is `defaults/main.yml`.  
```

Comment:
maybe you can display the project structure as well, then it is easier to follow if you mention specific files